USAGE.md

```diff
@@ -0,0 +1,48 @@
+# config-parse(1) -- screwdriver config parser
+
+## SYNOPSIS
+
+    config-parse [options] <screwdriver.yaml> <job-name> <build-number>
+
+## DESCRIPTION
+
+The config-parse utility parses a given `screwdriver.yaml` and writes out the list of steps to run as well as environment variables to set.
+
+It selects the steps and environment variables based on the specified `job-name` and `build-number`.  The `job-name` is the name specified in the `screwdriver.yaml` while the `build-number` is an incrementing number whose fractional part references which matrix permutation to run.
+
+For more information on build, please see the [domain model](https://screwdriver-cd.github.io/guide/architecture/domain/).
+
+## OPTIONS
+
+`-h, --help`
+    output usage information
+
+`-V, --version`
+    output the version number
+
+`-a, --artifactdir <directory>`
+    Specify an alternative artifact directory (default ./artifacts)
+
+## EXAMPLE
+
+To view the output from the main build, run something like the following:
+
+    config-parse ./screwdriver.yaml main 15.1
```

Comment:
Maybe change it to `view the output from the main build with run number 15...`  or something like that would help make this more clear?